cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2827,6 +2827,7 @@ Slice::Gen::InterfaceVisitor::visitInterfaceDefEnd(const InterfaceDefPtr& p)
     const string scoped = p->mappedScoped();
     const string name = p->mappedName();
 
+    // TODO this seems like a bug. We don't generate 'ice_id' etc for operation-less interfaces?
```

Comment:
This looks correct to me.

Note that this code generates dispatch, not ice_id etc. ice_id etc. are implemented by `ObjectI<T>`.

Reply:
Okay yes, I forgot that we would inherit the dispatch here.
Good to know!